content/en/tracing/version_tracking/_index.md

```diff
@@ -14,6 +14,75 @@ further_reading:
 
 This page describes specific ways you can use the `version` tag to better monitor deployments and service behavior. See the [Unified Service Tagging][1] docs for instructions for setting up the tags.
 
+## Using Version Tags on the Service Page
+
+{{< img src="tracing/version_tracking/ServicePage.png" alt="Versions on the Service Page"  style="width:100%;">}}
+
+On the Service page, if the `version` tag is available, the requests widget can be scoped to either of:
+
+- Total Requests by Version
+- Requests per second by Version
+
+The errors widget can be scoped to one of three options that involve the `version` tag:
+
+- Total Errors by Version
+- Errors per second by Version
+- % Error rate by Version
+
+All of these can be exported to dashboards and monitors.
+
+## Version Summary Table
+
+Services configured with version tags will have a `version` section on the Service page, immediately below the main service health graphs.  This table will show all versions of the service that were active during the selected time interval, with currently active services at the top.
+
```

Comment:
Do we normally use the future tense in docs? I would expect everything to be in the present tense. 
>Services configured with version tags have a `version` section on the Service page, immediately below the main service health graphs.  This section shows all versions of the service that were active during the selected time interval, with currently active services at the top.